pyproject.toml

```diff
@@ -13,7 +13,7 @@ types-requests = "^2.31.0.20240106"
 black = "^23.12.1"
 autoflake = "^2.2.1"
 isort = "^5.13.2"
-prediction-prophet = {git = "https://github.com/polywrap/predictionprophet", rev = "164ba3e20ace3b9f594cf9ba558807774dcd4fff"}
+prediction-prophet = {git = "https://github.com/polywrap/predictionprophet", rev = "d74a96456704dfc21cb049393123192facce91ac"}
```

Comment:
Sorry, why can't we do release or tag based instead of commit?

Reply:
Because Polywrap isn't doing either of it in their repository: https://github.com/polywrap/predictionprophet

But if you want, you can propose it to them, but I'd just go in with this PR and do that in another one.